lisa/tools/ping.py

```diff
@@ -72,6 +76,10 @@ def ping(
                 sudo=True,
             ).wait_result()
         if not ignore_error:
+            if self._dns_name_resolution_issue.findall(result.stdout):
```

Comment:
It doesn't need to find the special error message, any failure of ping should be reported. Just change the L83 to return the stdout, if it's not too many lines.

Reply:
update